### PR TITLE
add section about

### PR DESCRIPTION
**Pull request by Alisa-Bogatyryova**

### CODE REVIEW

File: src/partials/about.html [hunk 1]
```diff
@@ -1 +1,59 @@
...
+        />
+        <source
+          media="(max-width:767px)"
+          srcset="./images/mobile/cow_mobile.png 1x, ./images/mobile/cow@2x_mobile.png 2x"
+          type="image/png"
+        />
+        <img src="./images/mobile/cow_mobile.png" alt="Cow with milk" />
+      </picture>
+    </div>
+    <div class="about__discrp">
```
Comment (izachok) [inline src/partials/about.html]: about__descr

File: src/sass/layouts/_about.scss [hunk 2]
```diff
@@ -1,3 +1,172 @@
...
+    background-image: url(../images/desktop/bg-how-it-made_desktop.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+}
+.about__label {
+  margin-bottom: 15px;
+  font-family: var(--secondary-font-family);
+  font-style: normal;
+  font-weight: normal;
```
Comment (izachok) [inline src/sass/layouts/_about.scss]: font-style & font-weight normal are default values, no need to set them

File: src/sass/main.scss [hunk 3]
```diff
@@ -14,3 +14,4 @@
 @import './base/modals';
 
 @import './layouts/index';
+@import './layouts/about';
```
Comment (izachok) [inline src/sass/main.scss]: already included in @import './layouts/index';

**Review by izachok — CHANGES_REQUESTED** [review]

File: src/sass/main.scss (hunk 3 above)
Comment (Alisa-Bogatyryova) [reply src/sass/main.scss]: удалила

File: src/sass/layouts/_about.scss (hunk 2 above)
Comment (Alisa-Bogatyryova) [reply src/sass/layouts/_about.scss]: удалила

File: src/partials/about.html (hunk 1 above)
Comment (Alisa-Bogatyryova) [reply src/partials/about.html]: переименовала

**Review by izachok — APPROVED** [review]

**Pull request merged**